src/iuse.cpp

```diff
@@ -7874,9 +7878,9 @@ int iuse::ladder( player *p, item *, bool, const tripoint& )
     return 1;
 }
 
-int iuse::saw_barrel( player *p, item *, bool, const tripoint& )
+int iuse::saw_barrel( player *p, item *, bool ticking, const tripoint& )
```

Comment:
Why is the parameter here named differently? It's named `t` in the other `iuse` functions. It should be named the same in all of them, for consistency, as it means the same in all of them.